Documentation/apispec/YamlUtility.md

```diff
@@ -7,4 +6,4 @@ remarks: '*THIS* is remarks overriden in *MARKDWON* file'
 Expresses static methods for creating, moving, and enumerating through directories and subdirectories. This class cannot be inherited.
 To browse the .NET Framework source code for this type, see the [Reference Source](http://referencesource.microsoft.com/#mscorlib/system/io/directory.cs)
 
-Refer to @Microsoft.DocAsCode.EntityModel.ProjectLevelCache to see what to do with **Files**
\ No newline at end of file
+Refer to @"Microsoft.DocAsCode.EntityModel.ProjectLevelCache" to see what to do with **Files**
```

Comment:
is it a mandatory to add `"`, that would be a breaking change to `@` syntax definition